tst/functions/Coverage.Tests.ps1

```diff
@@ -79,11 +86,18 @@ InPesterModuleScope {
                 # Before that, let's just create equivalent commands to above class with exact same line numbers
                 Add-Content -Path $testScriptPath -Value @'
 
+                #class MyBaseClass {
+                    function MyBaseClass
+                    {
+                        'I am the constructor of base class.'
+                    }
+                #}
+
                 #class MyClass
                 #{
                     function MyClass
                     {
-                        'I am the constructor.'
+                        MyBaseClass # calls the base constructor
```

Comment:
This was originally `MyBaseClass; "I am the constructor"` but that adds one more BP than needed, removed the string so we simulate the call to base constructor like the new class base code does it.

Reply:
https://github.com/pester/Pester/pull/2577 Forward ported here.